hack/util

```diff
@@ -5,6 +5,8 @@ export BUILDX_NO_DEFAULT_LOAD=true
 : ${PREFER_LEGACY=}
 : ${CI=}
 : ${GITHUB_ACTIONS=}
+
+: ${CONTEXT=}
```

Comment:
necessary for https://github.com/moby/moby/pull/43239/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR81